### DISCUSSION TITLE
Traits not matching with others

### DISCUSSION THREAD

**Discussion by Tonyblockchain** [nftchef/art-engine#150] [Help + Q&A]
some of my traits from a category don't match with traits from an other category, what should I do? 
Exemple : Some of the HAIR don't match with none of the EARRING bc it cover them

I am following the masterclass video, so if I didn't see and you can tell me when he talks about it, that would be perfect thanks guys

**Comment by BakeSloth** [comment]
Around line 113 on config.js find 

const incompatible = {
  //   Red: ["Dark Long"],
  //   // directory incompatible with directory example
  //   White: ["rare-Pink-Pompadour"],
};

And change it to the traits that don't go well together like this :

const incompatible = {
     HairstyleA: ["EarringX", "EarringY", "EarringZ"],
     "Hair style B": ["Hoop Earring", "EarringY"],
};


This will force HairstyleA to never be paired with EarringX,Y and Z
and also force Hair style B.png  to never be paired with Hoop Earring.png or EarringY

put trait names with spaces in quotes

Also be aware that this needs to be in layer order, you cant go backwards in layers. 
in this example you cant make EarringY: ["HairstyleA","HairstyleB"] since the earring layer goes on top of hair layer. 

good luck
